packages/storefront/tests/vrt/specs/markdown.vrt.ts

```diff
@@ -17,25 +15,27 @@ test.describe('markdown', async () => {
       );
       await page.focus('a[href="https://designsystem.porsche.com/"]:not([title])');
       await page.setViewportSize({
```

Comment:
More of a general question: Why are we setting viewport width and height in every test or even in the middle of each test?
Seems like this could be done as part of device config https://playwright.dev/docs/emulation#viewport or before a test via `test.use`.
Setting height shouldn't matter in most cases when taking `fullPage` screenshots, right? https://playwright.dev/docs/screenshots#full-page-screenshots

Reply:
The height might be irrelevant as long as a node screenshot is taken instead of full page.
Experimenting with `test.use` to setup width before page navigation might be very good to increase reliability. As well as having e.g. a default width `viewportWidthM` defined in global playwright config.
I'll put it to the TODO list and will work on it in a follow up PR.